SOURCES/etc/xapi.d/plugins/raid.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from functools import wraps
+import sys
+import traceback
+import json
```

Comment:
Sort import

Reply:
We've never had an official coding style for python, but indeed maybe we can all agree on that rule